src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/RefreshOrchestratorTest.java

```diff
@@ -418,4 +515,18 @@ void refreshRoleAssignmentRecords_judicial() {
         assertEquals(HttpStatus.OK, response.getStatusCode());
         assertNotNull(response);
     }
+
+    @SuppressWarnings("unchecked")
+    @Test
+    @DisplayName("refreshRoleAssignmentRecords_Exception")
+    void refreshRoleAssignmentRecords_Exception() {
+        String msg = "Provided refresh job couldn't be retrieved.";
+
+        try {
+            sut.refresh(1L, TestDataBuilder.buildUserRequest());
+        } catch (UnprocessableEntityException e) {
+            assertEquals(msg, e.getLocalizedMessage().toString());
```

Comment:
verifying assert is in this way
Assertions.assertThrows(UnprocessableEntityException.class, () -> {
            sut.refresh(1L, TestDataBuilder.buildUserRequest());
        });

Reply:
Fixed the issue as suggested. Could you please review and provide sign-off